Source/Applications/SystemCenterNotification/Scripts/TSX/EmailTypes/Template.tsx

```diff
@@ -59,38 +57,44 @@ const Template = (props: IProps) => {
-                <div className="row">
-                    <div className="col">
-                        <h4>Template:</h4>
+        <div className="container-fluid d-flex h-100 flex-column" style={{ height: 'inherit' }}>
+            <div className="row" style={{ flex: 1, overflow: 'hidden' }}>
+                <div className="card" style={{ width: '100%', height: '100%' }}>
+                    <div className="card-header">
+                        <div className="row">
+                            <div className="col-12">
+                                <h4>Template:</h4>
+                            </div>
+                        </div>
+                    </div>
+                    <div className="card-body" style={{ paddingTop: 10, paddingBottom: 0, overflow: 'hidden' }}>
+                        <div className="row">
+                            <div className="col-12">
+                                <TextArea<EmailType> Record={email} Setter={setEmail} Field={'Template'} Rows={12} Label='' Valid={(r) => email.Template != null && email.Template.length > 0} />
+                            </div>
+                        </div>
+                    </div>
+                    <div className="card-footer">
+                        <div className="btn-group mr-2">
+                            <button className={"btn btn-primary" + (email.Template != null && email.Template.length > 0 && hasChanged ? '' : ' disabled')} type="submit"
+                                onClick={() => { if (email.Template != null && email.Template.length > 0 && hasChanged) dispatch(EmailTypeSlice.DBAction({ verb: 'PATCH', record: email })); }}
+                                data-tooltip='submit' onMouseEnter={() => setHover('submit')} onMouseLeave={() => setHover('none')}>Save Changes</button>
+                        </div>
+                        <div className="btn-group mr-2">
+                            <button className={"btn btn-default" + (hasChanged ? '' : ' disabled')} data-tooltip="clear"
+                                onClick={() => { setEmail(props.Record); setHasChanged(false); }}
+                                onMouseEnter={() => setHover('clear')} onMouseLeave={() => setHover('none')} >Clear Changes</button>
+                        </div>
+                        <ToolTip Show={(email.Template == null || email.Template.length == 0 || !hasChanged) && hover == 'submit'} Position={'top'} Theme={'dark'} Target={"submit"}>
+                            {!hasChanged ? <p> No changes made.</p> : null}
+                            {email.Template == null || email.Template.length == 0 ? <p> {CrossMark} A valid Template is required.</p> : null}
+                        </ToolTip>
+                        <ToolTip Show={hasChanged && hover == 'clear'} Position={'top'} Theme={'dark'} Target={"clear"}>
+                            {hasChanged ? <p> {Warning} Changes to Template will be discarded.</p> : null}
```

Comment:
```suggestion
                            <p> {Warning} Changes to Template will be discarded.</p>
```
I don't think you need this conditional since it only shows when hasChanged is true